common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1030,7 +1031,55 @@ def test_staff_can_edit_team_successfully(self):
         self.verify_and_navigate_to_edit_team_page()
 
         self.fill_create_or_edit_form()
-        self.create_or_edit_team_page.submit_form()
+
+        expected_events = [
+            {
+                'event_type': 'edx.team.changed',
+                'event': {
+                    'course_id': self.course_id,
+                    'team_id': self.team['id'],
+                    'field': 'country',
+                    'old': 'AF',
+                    'new': 'PK',
+                    'truncated': [],
```

Comment:
Technically, yes, but our longest field right now is `description`, which is limited to 300 characters, well below the truncation limit. Adding it to the `changed` event is mostly about being defensive in case someone adds a `TextField` into the model later.